repo/packages/S/spark/12/marathon.json.mustache

```diff
@@ -23,6 +30,15 @@
         "SPARK_SSL_PROTOCOL": "{{security.ssl.protocol}}",
         "SPARK_SSL_ENABLEDALGORITHMS": "{{security.ssl.enabledAlgorithms}}",
 {{/security.ssl.enabled}}
+{{#security.authn.enabled}}
```

Comment:
Rather than an 'enabled' flag, you could just do like @nickbp and trigger all of these sections on the absence/presence of `security.authn.service_account_secret_name`